src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -453,6 +459,11 @@ public void CompleteWrites()
 
     private unsafe int HandleEventStartComplete(ref START_COMPLETE data)
     {
+        if (NetEventSource.Log.IsEnabled())
```

Comment:
Could you also cover `QuicConnection` and `QuicListener` for a consistency.
Also could you log the varying arguments in a similar manner, like:
```
NetEventSource.Info(this, $"{this} Received event START_COMPLETE with {nameof(Status)}={data.Status}, {nameof(PeerAccepted)}={data.PeerAccepted}");
```

Reply:
QuicListener did not have anything interesting IMHO so I left is as is. I updated `QuicConnection` with same pattern.

Reply:
At least it has a new connection. And this is also about consistency.